.github/workflows/docs.yml

```diff
@@ -1,9 +1,9 @@
 name: Documentation
 
 on:
-  push:
-    branches: main
-  workflow_dispatch:
+  release:
+    types:
+      - published
```

Comment:
Is this now linked to tags or actual releases? I mean, in practice it should not make a difference because we have planned to always also create a release with each tag, but still, should we link it to tags instead such that we have it coupled to the thing that is actually pip-installable?

Reply:
This is not linked to tags. It does not make sense to link it to both tag and release, so we need to decide one.

I dont know for 100% sure, but they should be equivalent. In fact in the publish pipeline, you can see that both variants (trigger on tag or trigger on release) use the `${{  github.ref_name }}` which corresponds to the tag and is identical, otherwise the publishing wouldnt have worked as intended.

So it simply doesnt matter which variant we choose. The only thing is that you would prob be slightly more likely to fuck something up if you trigger on tag (eg small issue I caused when activated `publish all tags` in PyCharm etc), thats why I preferred on release trigger

Reply:
Ok, then let's keep as is and see what happens when we push the first tag 👍🏼